src/cmd/cli/command/cd.go

```diff
@@ -124,6 +136,20 @@ var cdListCmd = &cobra.Command{
 			return err
 		}
 
+		if remote {
+			loader := configureLoader(cmd)
+
+			projectName, err := cliClient.LoadProjectNameWithFallback(cmd.Context(), loader, provider)
```

Comment:
`cd ls` should not need a project name; there's an `if remote` right below, too.